website/docs/blog.mdx

```diff
@@ -625,6 +625,12 @@ Don't forget to delete the existing homepage at `./src/pages/index.js` or else t
 
 :::
 
+:::warning
+
+Don't forget to delete element of type doc in `docusaurus.config.js` in the navbar for example or else Docusaurus will crash.
```

Comment:
```suggestion
If you disable the docs plugin, don't forget to delete references to the docs plugin elsewhere in your configuration file. Notably, make sure to remove the docs-related navbar items.
```

That looks like a better wording to me?

I'd prefer to not have references to `docusaurus.config.js` considering the name is configurable and users can use TS